bin/migrate.php

```diff
@@ -27,27 +29,54 @@
 $io = new SymfonyStyle(new ArgvInput(), new ConsoleOutput());
 
 $inputDefinition = new InputDefinition([
-    new InputArgument('asset-family-code', InputArgument::REQUIRED),
-    new InputArgument('pim-path', InputArgument::REQUIRED)
+    new InputArgument('asset_family_code', InputArgument::REQUIRED),
+    new InputArgument('pim_path', InputArgument::REQUIRED),
+    new InputOption('--help', '-h', InputOption::VALUE_NONE, 'Display this help message'),
 ]);
 
+if (count(array_intersect($_SERVER['argv'], ['-h', '--help']))) {
+    $io->writeln(sprintf("<comment>Description:</comment>
+  Migrate an asset family in full-automatic scenario
+
+<comment>Usage:</comment>
+  %s
+
+<comment>Arguments:</comment>
+  <info>asset_family_code</info>  The asset family code to create
+  <info>pim_path</info>           The path to your PIM Enterprise Edition installation'
+
+<comment>Options:</comment>
+  <info>-h, --help</info>         Display this help message
+",
+        $_SERVER['argv'][0] . ' ' . $inputDefinition->getSynopsis(true)));
+
+    die(0);
+};
+
 try {
     $input = new ArgvInput(null, $inputDefinition);
     $input->validate();
-} catch (\Exception $e) {
-    $io->error(sprintf("Input format error: please use %s %s", 'migrate.php', $inputDefinition->getSynopsis(true)));
+} catch (RuntimeException $e) {
```

Comment:
Apparently we can now catch "Throwable" and it catches EVERY THING even PHP errors

Reply:
I don't want to catch EVERYTHING, I just want to catch the \Exception raised by the ->validate

Reply:
ouki